### PR TITLE
On boarding/nahuel gil/my trello app

### PR DESCRIPTION
**Pull request by NahuelGil98**

### CODE REVIEW

File: src/app/components/board-component/board-component.component.html [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+
+        </mat-tab>
+      </mat-tab-group>
+
+    </div>
+
+  <div>
+  </div>
+  <div>
+<!--    <mat-grid-list cols="4" rowHeight="2:2">-->
```
Comment (rjulianialtimetrik) [inline src/app/components/board-component/board-component.component.html]: Please simply remove the unused lines of code.

File: src/app/components/card-component/card-component.component.ts [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  name: string;
+}
+
+@Component({
+  selector: 'app-card-component',
+  templateUrl: './card-component.component.html',
+  styleUrls: ['./card-component.component.css']
+})
+export class CardComponentComponent implements OnInit {
+  @Input() title: string;
```
Comment (rjulianialtimetrik) [inline src/app/components/card-component/card-component.component.ts]: It's better if you define a model class, let's say, "Ticket" or "BoardItem" that contains your item's attributes. Then, you bind your inputs to said attributes in an instance of the model. makes up for clearer, self-contained and more reusable code.

File: src/app/components/card-component/card-component.component.ts [hunk 3]
```diff
@@ -0,0 +1,53 @@
...
+      width: '250',
+      data:  {cId: cardId, tId: tabId, cardTitle: this.title, cardDesc: this.description},
+    });
+    dialogRef.afterClosed().subscribe(result => {
+      console.log('The dialog was closed');
+      this.cardResult = result;
+      console.log(result);
+      // tslint:disable-next-line:triple-equals
+      if (result != 'undefined' || result != '') {
+        console.log(`Result : ${result} - cardId ${cardId} - tabId ${tabId}`);
```
Comment (rjulianialtimetrik) [inline src/app/components/card-component/card-component.component.ts]: might be a good idea to remove all console.log statements before pushing to the repo.

File: src/app/components/card-dialog/card-dialog.component.ts [hunk 4]
```diff
@@ -0,0 +1,30 @@
...
+import {MAT_DIALOG_DATA, MatDialogRef} from '@angular/material';
+import {DialogData} from '../card-component/card-component.component';
+
+@Component({
+  selector: 'app-card-dialog',
+  templateUrl: './card-dialog.component.html',
+  styleUrls: ['./card-dialog.component.css']
+})
+export class CardDialogComponent implements OnInit {
+  cardTitle = '';
```
Comment (rjulianialtimetrik) [inline src/app/components/card-dialog/card-dialog.component.ts]: Same as before, please define a "Card" model class and use it for your inputs.

File: src/app/components/column-comoponent/column-comoponent.component.html [hunk 5]
```diff
@@ -0,0 +1 @@
+<p>column-comoponent works!</p>
```
Comment (rjulianialtimetrik) [inline src/app/components/column-comoponent/column-comoponent.component.html]: Is this component supposed to be used in your app at all? if not, please remove it.

File: src/app/services/tabs.service.ts [hunk 6]
```diff
@@ -0,0 +1,90 @@
...
+
+  getTabs() {
+    return this.tabs;
+  }
+
+// Add a tab to an Array
+  addTab(tabTitle) {
+  console.log(` tab title: ${tabTitle}`);
+  if (tabTitle !== '' || tabTitle === undefined) {
+      this.newTab = {
```
Comment (rjulianialtimetrik) [inline src/app/services/tabs.service.ts:25]: It's better to have a model class for tab

File: src/app/services/tabs.service.ts [hunk 7]
```diff
@@ -0,0 +1,90 @@
...
+      this.tabs.push(this.newTab);
+      this.openSnackBar(`The tab ${tabTitle} was created successfully`, 'OK');
+    } else {
+      alert(`The name couldn't be empty`);
+    }
+  }
+// Add a card to a tab
+  addCard(card, tabId) {
+    this.newCard = {
+      id: uuid(),
```
Comment (rjulianialtimetrik) [inline src/app/services/tabs.service.ts]: it's better to have a model class for card

**Review by rjulianialtimetrik — CHANGES_REQUESTED** [review]

File: src/app/components/card-component/card-component.component.ts (hunk 2 above)
Comment (NahuelGil98) [reply src/app/components/card-component/card-component.component.ts]: Perfect, Thanks!

File: src/app/components/card-component/card-component.component.ts (hunk 3 above)
Comment (NahuelGil98) [reply src/app/components/card-component/card-component.component.ts]: Done, thanks!

File: src/app/components/board-component/board-component.component.html (hunk 1 above)
Comment (NahuelGil98) [reply src/app/components/board-component/board-component.component.html]: Done, Thanks !

File: src/app/services/tabs.service.ts (hunk 6 above)
Comment (NahuelGil98) [reply src/app/services/tabs.service.ts:25]: Ok, Thanks

File: src/app/services/tabs.service.ts (hunk 7 above)
Comment (NahuelGil98) [reply src/app/services/tabs.service.ts]: Ok, Thanks!

**Review by rjulianialtimetrik — APPROVED** [review]

**Pull request merged**